sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -1,19 +1,70 @@
+
+ `EVENT_HUBS_CONNECTION_STRING`  
+  The full connection string to the Event Hubs namespace, using the default shared access policy.
+    
+`EVENT_HUBS_NAMESPACE`  
+ The host name of the Event Hubs namespace, without the protocol or domain found in the connection string.
+    
+`EVENT_HUBS_RESOURCEGROUP`  
+ The name of the Azure resource group that contains the Event Hubs namespace
+   
+`EVENT_HUBS_SUBSCRIPTION`  
+ The identifier (GUID) of the Azure subscription to which the service principal belongs
+    
+`EVENT_HUBS_TENANT`  
+ The identifier (GUID) of the Azure Active Directory tenant that contains the service principal
+
+`EVENT_HUBS_CLIENT`  
+ The identifier (GUID) of the Azure Active Directory application that is associated with the service principal
+   
+`EVENT_HUBS_SECRET`  
+ The client secret (password) of the Azure Active Directory application that is associated with the service principal
+   
+To make setting up your environment easier, a [PowerShell script](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/assets/live-tests-azure-setup.ps1) is included in the repository and will create and/or configure the needed Azure resources.  To use this script, open a PowerShell instance and login to your Azure account using `Login-AzAccount`, then execute the script.   You will need to provide some information, after which the script will configure the Azure resources and then output the set of environment variables with the correct values for running tests.
+
+The simplest way to get started is to execute the script with your subscription name and then follow the prompts:
+
+```powershell
+./live-tests-azure-setup -SubscriptionName "<< YOUR SUBSCRIPTION NAME >>"
+```
+
+Help for the full set of parameters and additional information is available by specifying the `-Help` flag.
+
+```powershell
+./live-tests-azure-setup -Help
+```
+
+### Samples 
+
+In order to run the samples interactively, you'll need an Event Hubs namespace and an Event Hub with at least one partition.  Each sample will take a connection string and Event Hub name as parameters when executing, which can either be supplied directly as part of development or can be specified to the console application host in the `Samples` project, either using command line arguments or entered in response to prompts.
+
+## Development history
+
+For additional insight and context, the development, release, and issue history for the Azure Event Hubs client library is available in read-only form, in its previous location, the [Azure Event Hubs .NET repository](https://github.com/Azure/azure-event-hubs-dotnet).
```

Comment:
Missing 'Client' in the name of this link... should be `Microsoft Azure Event Hubs Client for .NET`?

Reply:
Nope, because this was the service team's former repository with service, client, management, and other code in there.    :)